CoreFindView/src/au/gov/asd/tac/constellation/views/find2/components/advanced/utilities/UseListInputWindow.java

```diff
@@ -0,0 +1,203 @@
+            splitLinesChoiceAction(newElement);
+        });
+
+        splitLinesButton.setOnAction(action -> {
+            splitLinesAction();
+        });
+
+        saveButton.setOnAction(action -> {
+            parentComponent.setSearchFieldText(getTextAreaText());
+            close();
+        });
+
+        cancelButton.setOnAction(action -> {
+            close();
+        });
+    }
+
+    /**
+     * Sets the UI content of the window
+     */
+    private void setContent() {
+        textArea.setText(parentComponent.getSearchFieldText());
+        splitLinesChoiceBox.getItems().addAll("Comma", "Tab", "Semi-colon", "String");
+        splitLinesChoiceBox.getSelectionModel().selectFirst();
+        chosenSplitLineOption = splitLinesChoiceBox.getSelectionModel().getSelectedItem();
+
+        splitLinesHbox.getChildren().addAll(splitLinesChoiceBoxLabel, splitLinesChoiceBox, splitLinesButton);
+        splitLinesHbox.setSpacing(10);
+
+        vbox.getChildren().addAll(textAreaLabel, textArea, splitLinesHbox);
+        vbox.setSpacing(10);
+        vbox.setPadding(new Insets(10));
+
+        buttonsHbox.getChildren().addAll(saveButton, cancelButton);
+        buttonsHbox.setSpacing(10);
+        buttonsHbox.setPadding(new Insets(0, 10, 10, 0));
+        buttonsBp.setRight(buttonsHbox);
+        bp.setBottom(buttonsBp);
+
+        bp.setCenter(vbox);
+        Scene scene = new Scene(bp);
+        scene.getStylesheets().add(IconSelector.class.getResource(DARK_THEME).toExternalForm());
+
+        setScene(scene);
+    }
+
+    /**
+     * This is called by the parent object when opening this window. It updates
+     * the textAreas text to be a formatted version of the parents textField
+     * text.
+     *
+     * This,Is,a,test = This\nIs\n\ntest
+     *
+     * @param text
+     */
+    public void updateText(String text) {
+        StringBuilder sb = new StringBuilder();
+        String[] splitText = text.split(",");
+        for (int i = 0; i < splitText.length; i++) {
+            sb.append(splitText[i]);
+            sb.append(i == splitText.length - 1 ? "" : "\n");
+        }
+        textArea.setText(sb.toString());
+    }
+
+    /**
+     * Gets the textAreas current text
+     *
+     * @return
+     */
+    public String getTextAreaText() {
+        return textArea.getText();
+    }
+
+    /**
+     * This is called when the split lines button is pressed. It splits the text
+     * areas text based on the splitLinesChoiceBox selection. For example if the
+     * text area contains This,Is,a,test and the user has split by commas
+     * selected. The text are will now contain This\nIs\n\ntest
+     */
+    private void splitLinesAction() {
+        String selectedSplitChoice = splitLinesChoiceBox.getSelectionModel().getSelectedItem();
+        String splitAt;
+        switch (selectedSplitChoice) {
+            case "Comma":
+                splitAt = ",";
```

Comment:
Use `SeparatorConstants` for these `splitAt` sets (there's one for semicolon and tab as well)